src/agent/sqlite_manager/include/sqlite_manager.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+namespace sqlite_manager
+{
+    class SQLiteManager
+    {
+    public:
+        SQLiteManager();
+        ~SQLiteManager() {};
```

Comment:
Empty constructors and destructors are not necessary for an empty class. Also, no need for semicolon after `{}`.

Reply:
Done